README.md

```diff
@@ -86,6 +86,11 @@ It adds only a few lightweight `require` statements to the bootstrap process
 to support all polyfills. Implementations are then loaded on-demand when
 needed during code execution.
 
+If your project requires a minimum PHP version it is advisable to add polyfills
+for lower PHP versions to the `replace` section of your `composer.json`. 
+This removes any overhead from these polyfills as they are no longer part of your project.
+The same can be done for polyfills for extensions that you require.
```

Comment:
What about adding a `composer.json` snippet to illustrate how the `replace` section would look like for a give version of PHP?